data/assets/03_global.css

```diff
@@ -27,29 +27,36 @@ h3 {
 
 /* Page top horizontal bar */
 .main_header {
+  display: inline-block;
+  width: 100%;
   font-size: var(--font-size-large);
   background: var(--blue-0);
   color: var(--white-0);
-  text-align: center;
-  padding-top: 20px;
-  padding-bottom: 40px;
+  padding: 5px 20px;
 }
 
 .header_title {
-  width: 20%;
   float: left;
+  margin-right: 20px;
```

Comment:
Fixed sized gaps (defined in absolute units) have an increased chance to not work on smaller screens. Relative sizing is almost always better.